<ui version="4.0" >
 <class>LapDialog</class>
 <widget class="QDialog" name="LapDialog" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>242</width>
    <height>355</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>0</hsizetype>
    <vsizetype>0</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>242</width>
    <height>355</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>242</width>
    <height>355</height>
   </size>
  </property>
  <property name="font" >
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle" >
   <string>Select options</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>false</bool>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="verticalLayout" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>200</y>
     <width>221</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QCheckBox" name="checkBox" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>preview</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxBlur" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>blur</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QCheckBox" name="checkBoxNeg" >
          <property name="text" >
           <string>negative</string>
          </property>
          <property name="checked" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxNor" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
          <property name="text" >
           <string>normalize</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>5</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize" >
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font" >
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="title" >
         <string>mask</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignLeading</set>
        </property>
        <widget class="QRadioButton" name="radioButton4" >
         <property name="geometry" >
          <rect>
           <x>10</x>
           <y>20</y>
           <width>101</width>
           <height>23</height>
          </rect>
         </property>
         <property name="toolTip" >
          <string> 0 -1 0
-1 4 -1
 0 -1 0</string>
         </property>
         <property name="text" >
          <string>"4"</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton8" >
         <property name="geometry" >
          <rect>
           <x>120</x>
           <y>20</y>
           <width>101</width>
           <height>23</height>
          </rect>
         </property>
         <property name="toolTip" >
          <string>-1 -1 -1
-1  8 -1
-1 -1 -1</string>
         </property>
         <property name="text" >
          <string>"8"</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton" >
        <property name="text" >
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="okButton" >
        <property name="text" >
         <string>OK</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="PreviewLabel" name="previewLabel" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>181</height>
    </rect>
   </property>
   <property name="cursor" >
    <cursor>9</cursor>
   </property>
   <property name="frameShape" >
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="textFormat" >
    <enum>Qt::AutoText</enum>
   </property>
   <property name="pixmap" >
    <pixmap/>
   </property>
   <property name="scaledContents" >
    <bool>false</bool>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter</set>
   </property>
   <property name="textInteractionFlags" >
    <enum>Qt::NoTextInteraction</enum>
   </property>
   <property name="buddy" >
    <cstring>checkBox</cstring>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PreviewLabel</class>
   <extends>QLabel</extends>
   <header>src/previewlabel.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>checkBox</tabstop>
  <tabstop>checkBoxBlur</tabstop>
  <tabstop>checkBoxNeg</tabstop>
  <tabstop>checkBoxNor</tabstop>
  <tabstop>radioButton4</tabstop>
  <tabstop>radioButton8</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources>
  <include location="application.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>LapDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>LapDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
